README.md

```diff
@@ -64,3 +64,5 @@ If you discover a security vulnerability within Laravel, please send an e-mail t
 ## License
 
 The Laravel framework is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).
+
+test juan
```

Comment:
please remove the test string again

Reply:
It has already been removed.

Reply:
ok, but if i merge this pull request it would be added again, you either have to merge viktorias branch into yours or remove it in your branch